saleor/order/calculations.py

```diff
@@ -303,12 +89,14 @@ def fetch_order_prices_if_expired(
                 "tax_rate",
             ],
         )
+        if info:
+            OrderLinesByOrderIdLoader(info.context).clear(order.id)
```

Comment:
I am not conviced that mixing the business logic with the API layer is correct approach. 
Can you provide more context, when the data was out-of-date? Was it working correctly previously? If yes, then why it doesn't work now.

Reply:
You are right, it is not needed anymore.